### PR TITLE
Remove vestigial references to signature

### PR DESCRIPTION
**Pull request by bifurcation**
The signature mechanism was removed from SFrame some time ago.  This PR removes a few remaining references to it.

### CODE REVIEW

**Comment by youennf** [comment]
Seems good but needs rebasing

File: draft-ietf-sframe-enc.md [hunk 1]
```diff
@@ -556,8 +553,8 @@ ephemeral symmetric keys for a specific call.
 
 If the participants in a call have a pre-existing E2E-secure channel, they can
 use it to distribute SFrame keys.  Each client participating in a call generates
-a fresh encryption key and optionally a signing key pair.  The client then uses
-the E2E-secure channel to send their encryption key and signing public key to
+a fresh encryption key.  The client then uses
```
Comment (youennf) [inline draft-ietf-sframe-enc.md:556]:
```suggestion
a fresh encryption key. The client then uses
```

File: draft-ietf-sframe-enc.md [hunk 2]
```diff
@@ -578,9 +575,9 @@ old key may be kept for some time to allow for out-of-order delivery, but should
 be deleted promptly.
 
 If a new participant joins mid-call, they will need to receive from each sender
-(a) the current sender key for that sender, (b) the signing key for the sender,
-if used, and (c) the current KID value for the sender.  Evicting a participant
-requires each sender to send a fresh sender key to all receivers.
+(a) the current sender key for that sender and (b) the current KID value for the
+sender.  Evicting a participant requires each sender to send a fresh sender key
```
Comment (youennf) [inline draft-ietf-sframe-enc.md:579]:
```suggestion
sender. Evicting a participant requires each sender to send a fresh sender key
```

File: draft-ietf-sframe-enc.md [hunk 3]
```diff
@@ -539,11 +537,10 @@ def AEAD.Decrypt(key, nonce, aad, ct):
 # Key Management
 
 SFrame must be integrated with an E2E key management framework to exchange and
-rotate the keys used for SFrame encryption and/or signing.  The key management
+rotate the keys used for SFrame encryption.  The key management
```
Comment (youennf) [inline draft-ietf-sframe-enc.md:540]:
```suggestion
rotate the keys used for SFrame encryption. The key management
```

**Review by youennf — APPROVED** [review]

**Review by murillo128 — APPROVED** [review]

**Comment by murillo128** [comment]
@youennf @bifurcation I think we are ok to merge this PR?

**Pull request merged**